src/main/java/uk/gov/hmcts/darts/audit/service/AuditService.java

```diff
@@ -11,8 +11,6 @@
 
 public interface AuditService {
 
-    void recordAuditRequestAudio(AuditActivityEnum activity, Integer userAccountEntity, Integer courtCase);
-
     @Transactional
```

Comment:
This `@Transactional` can be removed from the interface, since we have it on the implementing class which I think is more normal

Reply:
Done